### ISSUE TITLE
Rename <firebase-storage-multiupload> to <firebase-storage-upload>

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [firebase/polymerfire#262]
What about renaming `<firebase-storage-multiupload>` to just `<firebase-storage-upload>`?

**Comment by FluorescentHallucinogen** [comment]
@tjmonsi @mbleigh PTAL.

**Comment by mbleigh** [comment]
I'd like to leave the door open for a simpler element that is dedicated to uploading a single file. That being said, I'm trying to open this up more to the community since my time to maintain is super-limited, so if @tjmonsi wants to rename I wouldn't object (though since it's been released, this would be a breaking change).

**Comment by tjmonsi** [comment]
@mbleigh @FluorescentHallucinogen yeah it would be a breaking change. I think I can do a "similar" file that is extended from firebase-storage-multiupload but that would just be for Polymer 2.0 and above.

@mbleigh my doors are open to help maintain if you still need help :)
